File: exercises/Coding - Exercício3 - Igor Barbosa - 29.08.25.py
print("Qual a sua nota na primeira avaliação? ")
n1 = float(input())
print("Qual a sua nota na segunda avaliação: ")
n2 = float(input())
print("Qual a sua nota no seminário? ")
n3 = float(input())
media = n1 + n2 + n3 / 3
print("A sua média é: " + str(media))
if media >= 7:
    print("Aprovado, muito bem! ")
else:
    print("Reprovado, se esforce um pouco mais ... ")

File: exercises/Coding - Exercício3 - Igor Barbosa - 29.08.25.js
function main() {
    var n1, n2, n3, media;

    window.alert("Qual a sua nota na primeira avaliação? ");
    n1 = Number(window.prompt('Enter a value for n1'));
    window.alert("Qual a sua nota na segunda avaliação: ");
    n2 = Number(window.prompt('Enter a value for n2'));
    window.alert("Qual a sua nota no seminário? ");
    n3 = Number(window.prompt('Enter a value for n3'));
    media = n1 + n2 + n3 / 3;
    window.alert("A sua média é: " + media);
    if (media >= 7) {
        window.alert("Aprovado, muito bem! ");
    } else {
        window.alert("Reprovado, se esforce um pouco mais ... ");
    }
}
